server/backend/api/app/routers/router_application.py

```diff
@@ -68,16 +68,18 @@ def get_fam_application_roles(
 def get_fam_application_user_role_assignment(
     application_id: int,
     db: Session = Depends(database.get_db),
+    requester: Requester = Depends(get_current_requester)
 ):
-    """gets the roles associated with an application
-
-    :param application_id: application id
-    :param db: database session, defaults to Depends(database.get_db)
+    """
+    gets the roles assignment associated with an application
     """
     LOGGER.debug(f"Loading application role assigments for application_id: {application_id}")
     app_user_role_assignment = crud_application.get_application_role_assignments(
-        db=db, application_id=application_id
+        db=db,
+        application_id=application_id,
+        requester=requester
```

Comment:
Hi Ian, just a thought, I wonder if you could just pass requester.business_guid to the curd method, instead of the whole requester? then in the crud method, we just check when business_guid is not null. And when writing tests, we don't need to add the get current requester method anymore. 

I checked our admin management apis, for example `admin_management/api/app/routers/router_access_control_privilege.py` just pass the requester's cognito user id, not the whole requester from router to service method.

Reply:
Hi Catherine, I would not rely upon "**business_guid**" to check on null or not in crud method as this is not the identifier for the requester. The Requester is a system object derived from the user's cognito user id, it is more reliable and ok to pass throughout the system. If we choose not passing the whole requester object, it would still need to pass several things (cognito_user_id, business_guid and user_type) for code to be easier to query.